src/scripts/main.js

```diff
@@ -1,9 +1,48 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.body;
+
+  const div = getMessage(title, description, type);
+
+  div.style.top = `${posTop}px`;
+  div.style.rigth = `${posRight}px`;
+
+  body.append(div);
+
+  setTimeout(() => div.remove(), 2000);
 };
 
+function getMessage(title, description, type) {
+  const msgBlock = document.createElement('div');
+
+  msgBlock.classList.add('notification');
+
+  const titleMsg = document.createElement('h2');
+
+  titleMsg.classList.add('title');
+  titleMsg.textContent = `${title}`;
+
+  const desc = document.createElement('p');
+
+  desc.innerText = `${description}`;
+  msgBlock.append(titleMsg, desc);
+
+  switch (type) {
+    case 'success':
+      msgBlock.className += ` ${type}`;
+      break;
+    case 'warning':
+      msgBlock.className += ` ${type}`;
+      break;
+    default:
+      msgBlock.className += ` ${type}`;
+      break;
+  }
```

Comment:
looks like you use the same solution for all cases

Reply:
you can get rid of `switch case` statement

Reply:
switch case statement removed